### PR TITLE
Added push endpoint to parse urls out of json submission

### PR DESCRIPTION
**Pull request by jamielitt-guitar**
Added in "body-parser" dependency which seems to be part of express anyway which is interesting.
Added in push endpoint so we can send JSON to it such as:

```
{
    "urls" : [
            "https://www.confused.com",
            "https://www.confused.com/car-insurance"
    ]
}
```

Tested in Postman and it seems to work

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,6 +14,22 @@ server.get('/helloworld', (req, res) => {
     res.send('Hello World - first initial dev')
 })
 
+// Endpoint for submitting a json request
+server.post('/inspecturls', jsonParser, (req, res) => {
+    
+    // Convert boduy to JSON
+    var inspector = req.body;
+    console.log("JSON input parsed into an object");
```
Comment (jamielitt) [inline index.js:22]: Possibly don't need this anymore

File: index.js [hunk 2]
```diff
@@ -11,6 +14,22 @@ server.get('/helloworld', (req, res) => {
...
+    
+    // Convert boduy to JSON
+    var inspector = req.body;
+    console.log("JSON input parsed into an object");
+
+    // Output to console window for now, we will do something different with it later
+    console.log(JSON.stringify(inspector));
+    res.send('JSON should have been output to console window');
+
+    inspector.urls.forEach(element => {
```
Comment (jamielitt) [inline index.js:28]: Quick comment above this just to say why for now

File: index.js (hunk 1 above)
Comment (jamielitt-guitar) [reply index.js]: Removed

File: index.js (hunk 2 above)
Comment (jamielitt-guitar) [reply index.js:24]: Done

File: index.js [hunk 3]
```diff
@@ -11,6 +14,22 @@ server.get('/helloworld', (req, res) => {
...
+// Endpoint for submitting a json request
+server.post('/inspecturls', jsonParser, (req, res) => {
+    
+    // Convert boduy to JSON
+    var inspector = req.body;
+    console.log("JSON input parsed into an object");
+
+    // Output to console window for now, we will do something different with it later
+    console.log(JSON.stringify(inspector));
+    res.send('JSON should have been output to console window');
```
[the remark this answers is not in the document]
Comment (jamielitt-guitar) [reply index.js]: Removed

**Review by jamielitt — APPROVED** [review]

**Pull request merged**